tutorials/IBC-connection.md

```diff
@@ -0,0 +1,283 @@
+# IBC connection tutorial
+
+In this tutorial, we'll learn how to use [relayer](https://github.com/cosmos/relayer) to
+create a IBC connection between [GM world](/tutorials/gm-world) rollup and a osmosis local testnet.
+
+## 💻 Pre-requisites
+
+### Software requirement
+
+* Docker running on your machine
+* Go version >= 1.21.0
+
+## Run a GM rollup chain
+
+Before you can create IBC connection, you need to start a
+local-celestia-devnet instance in a separate terminal:
+
+```bash
+docker run -t -i --platform linux/amd64 -p 26650:26650 -p 26657:26657 -p 26658:26658 -p 26659:26659 -p 9090:9090 ghcr.io/rollkit/local-celestia-devnet:v0.12.6
+```
+
+And we start the GM chain by using this script :
+
+```sh
+wget https://raw.githubusercontent.com/rollkit/docs/main/scripts/gm/init-local.sh
+sh init-full-node.sh
+```
+
+## Run your local-osmosis-testnet
+
+### Install osmosis binary
+
+```sh
+git clone https://github.com/osmosis-labs/osmosis
+cd osmosis
+git checkout v21.0.2
```

Comment:
`v21.0.2` seems to be missing?

Reply:
I misremmember the version name, it should be `v21.0.1`

Reply:
fixed !